### DISCUSSION TITLE
How to change Edit app to my IDE

### DISCUSSION THREAD

**Discussion by QuAzI** [gitextensions/gitextensions#11929] [Q&A] [answered]
Is there any way to use VSCode/notepad++/vim/current instance of Rider for this option?
![image](https://github.com/user-attachments/assets/88791649-09f8-4c2d-a842-780c3ee9f301)

**Comment by pmiossec** [comment]
No, I think not.

Your option is to use 'Open with...' or use drag& drop

**Comment by mstv** [comment, marked as answer]
You could create a user script like:

```
cmd /c if "{SelectedRelativePaths}" == "" ( code . ) else ( code . --goto {{SelectedRelativePaths}}:{LineNumber} )
```

and assign a hotkey to it. You could even unassign the hotkey `F4` from "EditFile" and assign it to the script.

![image](https://github.com/user-attachments/assets/59fc1f55-b1ca-471e-a53b-63dd267faf8f)

![image](https://github.com/user-attachments/assets/766d6c6a-03b4-4387-b97b-abb24a727868)

![image](https://github.com/user-attachments/assets/637658d2-09a5-4d60-8c63-d6c005f8f259)
